app/views/shared/_footer.html+new_navigation.erb

```diff
@@ -0,0 +1 @@
+<%= render 'govuk_component/document_footer', @content_item.document_footer_for_new_navigation %>
```

Comment:
We've been gradually moving as much logic into presenters as we can, so that templates are simplified for consolidation.